File: Dev/Assets/Scripts/Config/export_cs_model/PatternsPowerConfig.cs
//表名: 牌型倍率, 字段描述：_key:编号, _name:名称, _commision:倍率,  -->
using System;
using System.Collections.Generic;
using Mono.Xml;
using System.Security;
using Utils;

public class PatternsPowerConfigItem:ConfigDataItemBase{
//	private System.Object _key;
	/// <summary>
	/// 编号
	/// </summary>
	public string key { get { return (string)_key; } }
	private System.Object _name;
	/// <summary>
	/// 名称
	/// </summary>
	public string name { get { return (string)_name; } }
	private System.Object _commision;
	/// <summary>
	/// 倍率
	/// </summary>
	public int commision { get { return (int)_commision; } }

	public override void Parse(SecurityElement element) {
		_key = element.Attribute("key");base["key"]=_key;
		_name = element.Attribute("name");base["name"]=_name;
		_commision = Convert.ToInt32(element.Attribute("commision"));base["commision"]=_commision;
	}
}

public class PatternsPowerConfig : ConfigDataBase<PatternsPowerConfigItem> {
	public PatternsPowerConfig(){
		_fileName = "export_xml/patterns_power_config.bytes";
	}
}

File: Dev/Assets/luaScripts/config/data/PatternsPowerConfig.lua
--表名: 牌型倍率, 字段描述：_key:编号, _name:名称, _commision:倍率, 
local M = {}
M["1"] = {key = "1", name = "五小牛", commision = "8", }
M["2"] = {key = "2", name = "五花牛", commision = "5", }
M["3"] = {key = "3", name = "四炸", commision = "4", }
M["4"] = {key = "4", name = "牛牛", commision = "3", }
M["5"] = {key = "5", name = "牛9", commision = "2", }
M["6"] = {key = "6", name = "牛8", commision = "2", }
M["7"] = {key = "7", name = "牛7", commision = "2", }
M["8"] = {key = "8", name = "牛6", commision = "1", }
M["9"] = {key = "9", name = "牛5", commision = "1", }
M["10"] = {key = "10", name = "牛4", commision = "1", }
M["11"] = {key = "11", name = "牛3", commision = "1", }
M["12"] = {key = "12", name = "牛2", commision = "1", }
M["13"] = {key = "13", name = "牛1", commision = "1", }
M["14"] = {key = "14", name = "没牛", commision = "1", }
LuaConfigMgr.PatternsPowerConfigLen = 14
LuaConfigMgr.PatternsPowerConfig = M